CMakeLists.txt

```diff
@@ -0,0 +1,127 @@
+cmake_minimum_required(VERSION 3.23)
+project(
+    cppfront
+    LANGUAGES CXX
+    VERSION 0.1.0
+)
+
+option(
+    CPPFRONT_INSTALL_RULES "Enable install rules for cppfront" "${PROJECT_IS_TOP_LEVEL}"
```

Comment:
Any reason why the built-in variable is not used?

https://cmake.org/cmake/help/latest/variable/CMAKE_SKIP_INSTALL_RULES.html

Reply:
Because of this: https://gitlab.kitware.com/cmake/cmake/-/issues/22561

Reply:
Oh you mean skipping by checking... yes that is a better idea, thanks.